kobo/apps/subsequences/utils/deprecation.py

```diff
@@ -79,6 +79,13 @@ def qpath_to_xpath(qpath: str, asset: 'Asset') -> str:
         if '$qpath' in row and '$xpath' in row and row['$qpath'] == qpath:
             return row['$xpath']
 
+    # If qpath relates to an existing question on the asset, skip this field
+    # by returning an empty string
+    dashed_qpath = qpath.replace('/', '-')
+    for known_col in asset.known_cols:
+        if known_col.startswith(dashed_qpath):
+            return ''
+
```

Comment:
Is it possible to create a unit test for that? 

Maybe consider rephrasing the comment to make it clearer that this typically happens when a question is renamed in the form, and `qpath` no longer matches any existing XPath. That might help others better understand the context later on.